waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -803,9 +805,17 @@
         ;; delay iff the log-bucket-url setting was given the scheduler config.
         log-bucket-sync-secs (if log-bucket-url (:log-bucket-sync-secs context) 0)
         total-sigkill-delay-secs (+ pod-sigkill-delay-secs log-bucket-sync-secs)
+        has-reverse-proxy? (contains? env (:reverse-proxy-flag proxy-options))
```

Comment:
We should destructure `reverse-proxy-flag` and `reverse-proxy-offset`.
`{:keys [reverse-proxy-flag reverse-proxy-offset]} proxy-options`

Reply:
put in separate PR